File: go/1.go
package main

import "fmt"

func isMult(i int) bool {
    return i % 3 == 0 || i % 5 == 0
}

func main() {
    sum := 0
    for i := 0; i < 1000; i++ {
        if isMult(i) {
            sum += i
        }
    }
    fmt.Printf("%d\n", sum)
}

File: c/1.c
// finds sum of all multiples of 3 and 5 under 1000

#include <stdio.h>

int main(int argc, char** argv) {
    int sum = 0;
    int i;

    for (i = 1; i < 1000; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }

    printf("%d\n", sum);

    return 0;
}
